boards/shields/link_board_can/link_board_can.overlay

```diff
@@ -6,15 +6,12 @@
 
 &spi1 {
 	status = "okay";
-	cs-gpios = <&gpio1 3 0>;
-	sck-pin = <36>;
-	mosi-pin = <37>;
-	miso-pin = <38>;
+	cs-gpios = <&arduino_header 16 0>; /* D10 */
```

Comment:
I would not change it. This was a prototype, distributed to only one person, and it is not an Arduino connector shield. I will remove this shield soon as it is deprecated and the successor will be available after EW2020. Please drop this commit.

Reply:
OK, changes removed.  This is very confusing: if this is not an Arduino-compatible shield, it shouldn't be referencing the `arduino_spi` label.  Perhaps this should just be removed since it's for an undocumented prototype.  If for some reason that can't be done now, please make sure its replacement uses a different connector name.

Reply:
Agreed with @pabigot that we should either remove this is it was a prototype and not available to most people or at least clean it up to not use arduino references.

Reply:
Why send a prototype support for the Zephyr Project? The internal prototype for one customer doesn't seems to belong there.

Under development SoC or like, if later will be renamed for the final product name works fine as the code is reviewed and a future renamed is effective but one customer prototype belongs to the customer fork.

Reply:
> Why send a prototype support for the Zephyr Project? The internal prototype for one customer doesn't seems to belong there.
> 
> Under development SoC or like, if later will be renamed for the final product name works fine as the code is reviewed and a future renamed is effective but one customer prototype belongs to the customer fork.

Support was inserted by a contributor. The product has been discontinued and will be replaced by a successor. What does it concern you and whats is the point of serving here?

Reply:
I submitted the shield as a demonstrator on how to integrate the MCP2515, since there is no other board with this specific controller on it. @karstenkoenig and I use this prototype to test the driver implementation. We kind of use it as the reference.